product_inventory_service/plugins.sbt

```diff
@@ -0,0 +1,17 @@
+// The Play plugin
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.7")
+
+// The sbt native packager - needed for docker builds
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.8.0")
+
+// For code coverage test
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
+
+// For checkstyle
+addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
+
+// For formatting Scala source code
+addSbtPlugin("com.lucidchart" % "sbt-scalafmt" % "1.16")
```

Comment:
WTF is this?
`addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.2")` is official one